controllers/locker_controller.py

```diff
@@ -0,0 +1,118 @@
+import csv
+import re
+from typing import List
+from models import Locker
+from constants import LOCKER_PATH, MAX_LOCKERS
+from file_handler import read_csv, write_csv
+
+class LockerSystem:
+    def __init__(self):
+        self.lockers = self.load_lockers()
+    
+    def load_lockers(self) -> List[Locker]:
+        """사물함 데이터를 로드합니다."""
+        try:
+            # 파일이 비어있거나 헤더만 있는 경우 빈 리스트 반환
+            with open(LOCKER_PATH, newline='', encoding='utf-8') as csvfile:
+                reader = csv.DictReader(csvfile)
+                data = list(reader)
```

Comment:
read_csv 쓰는 걸로 바꾸시면 될 것 같습니다!

```suggestion
            data = read_csv(LOCKER_PATH)
```